### PR TITLE
Added extend and pointer logic for AENS names

### PR DESCRIPTION
**Pull request by vmangelovv**

### CODE REVIEW

File: src/store/actions.js [hunk 1]
```diff
@@ -293,7 +293,7 @@ export default {
                 pending: true,
                 owner: tx.accountId,
               })))(),
-          (async () => uniqBy(await (await fetch(`${middlewareUrl}/middleware/names/reverse/${publicKey}`)).json(), 'name'))(),
+          // (async () => uniqBy(await (await fetch(`${middlewareUrl}/middleware/names/reverse/${publicKey}`)).json(), 'name'))(),
```
Comment (mradkov) [inline src/store/actions.js]: You can just remove this. We have git history - no need to keep commented code.

File: package.json [hunk 2]
```diff
@@ -26,95 +26,95 @@
...
-    "@ledgerhq/hw-transport-u2f": "^4.72.1",
+    "@ledgerhq/hw-transport-u2f": "^4.78.0",
     "@stamp/it": "^1.1.0",
     "@vue/cli-plugin-e2e-cypress": "^3.12.0",
     "@vue/cli-service": "^3.12.0",
     "@zxing/library": "^0.15.2",
     "aeternity-tokens": "^1.0.2",
-    "argon2-browser": "^1.5.2",
-    "axios": "^0.19.0",
+    "argon2-browser": "^1.13.0",
```
Comment (mradkov) [inline package.json]: Last time there was a problem with argon2 update, does this work as intended or is breaking the build. 

Also are we still using argon2 somewhere after the switch to web crypto #219 ?

File: src/popup/router/pages/NamingSystem.vue [hunk 3]
```diff
@@ -219,16 +250,25 @@ export default {
...
       this.byBid = false;
       this.bySoonest = false;
+      this.byMine = false;
     },
     filterByBid() {
       this.byBid = true;
       this.byCharLength = false;
       this.bySoonest = false;
+      this.byMine = false;
     },
```
Comment (mradkov) [inline src/popup/router/pages/NamingSystem.vue]: We can optimize these filters by having a single method with a switch or some enum?

File: src/popup/router/pages/NamingSystem.vue [hunk 4]
```diff
@@ -51,23 +48,15 @@
...
-            <ae-filter-item class="au-filter notround" @click.native="filterBySoonest" :active="bySoonest ? true : false">
-              {{ $t('pages.namingSystemPage.filterBySoonest') }}
-            </ae-filter-item>
-            <ae-filter-item class="au-filter notround" @click.native="filterByCharLength" :active="byCharLength ? true : false">
-              {{ $t('pages.namingSystemPage.filterByCharLength') }}
-            </ae-filter-item>
-            <ae-filter-item class="au-filter notround" @click.native="filterByBid" :active="byBid ? true : false">
-              {{ $t('pages.namingSystemPage.filterByBid') }}
-            </ae-filter-item>
+            <ae-filter-item class="au-filter notround" @click.native="filterType = 'mine'" :active="filterType == 'mine' ? true : false"> {{ $t('pages.namingSystemPage.filterByMine') }} </ae-filter-item>
```
Comment (petbaik) [inline src/popup/router/pages/NamingSystem.vue]: We can leave the expression just with `:active="filterType === 'mine'`, because it returns boolean

**Pull request merged**